cdap-ui/app/cdap/components/AbstractWidget/FormInputs/TextBox/index.tsx

```diff
@@ -75,5 +75,14 @@ const TextBox: React.FC<ITextBoxProps> = ({
   );
 };
 
-export default React.memo(TextBox);
+const TextBox = React.memo(TextBoxComponent);
+
 (TextBox as any).propTypes = WIDGET_PROPTYPES;
+(TextBox as any).getWidgetAttributes = () => {
+  return {
+    placeholder: { type: 'string', required: false },
```

Comment:
Should default get added to ITextBoxWidgetProps?

Reply:
Done